network/serde/jsonrpc_test.go

```diff
@@ -24,11 +24,12 @@ func TestJsonRpcSerializeResponse(t *testing.T) {
 	rpc := JsonRpc{}
 	data, err := rpc.Serialize(&netproto.Message{
```

Comment:
Could we nest `network` code be under the `rpc` directory? 

Longer term, if there is common / reused code between the rpc networking and the p2p networking, we could have a top level `network` package.